torchbenchmark/score/compute_score.py

```diff
@@ -7,32 +7,137 @@
 import math
 import sys
 import os
+import re
 import yaml
+import importlib
 
 from tabulate import tabulate
+from pathlib import Path
+from collections import defaultdict
 
-SPEC_FILE_DEFAULT = "score.yml"
+SPEC_FILE_DEFAULT = "torchbenchmark/score/score.yml"
```

Comment:
these kind of hardcode paths should be changed to something based on __file__ probably.  otherwise it can break.

Path(__file__).parent() / "score.yml"
does that work? not sure if .parent() exists

Reply:
Fixed now.